ci/deploy-deb.sh

```diff
@@ -12,6 +12,20 @@ for release in ${DEBIAN_RELEASES[@]} ${UBUNTU_RELEASES[@]}; do
   reprepro -A arm64 remove $release trivy
 done
 
+rm -r conf
+mkdir conf
+
+for release in ${UBUNTU_RELEASES[@]}; do
+  echo "Origin: knqyf263.github.io/trivy-repo/deb
```

Comment:
We forgot to replace with aquasecurity.

```suggestion
  echo "Origin: aquasecurity.github.io/trivy-repo/deb
```

Reply:
updated